src/Version.cpp

```diff
@@ -34,28 +35,36 @@
 namespace espressopp {
 
   Version::Version() {
-	  name          = "ESPResSo++";
-      major         = MAJORVERSION;
-	  minor         = MINORVERSION;
-	  patchlevel    = PATCHLEVEL;
-	  gitrevision    = gitversion;
-      boostversion  = BOOST_LIB_VERSION;
-	  date          = __DATE__;
-	  time          = __TIME__;
+    name_          = "ESPResSo++";
+    major_         = MAJORVERSION;
+    minor_         = MINORVERSION;
+    patchlevel_    = PATCHLEVEL;
+    gitrevision_    = gitversion;
+    boostversion_  = BOOST_LIB_VERSION;
+    date_          = __DATE__;
+    time_          = __TIME__;
```

Comment:
Sure, committing only trailing space is not a good idea. But the rule 'touched-owned' really improve the style of code.
I think that running one-time such tool without any clear style code rule will not make any difference in long term developing. That's rather a politics and project organization. E.g. allow obsolete code to be on the `master` branch and so on.